source/patterns/01-molecules/figure/figure.twig

```diff
@@ -32,6 +38,7 @@
           {%- endwith -%}
 
           {%- with {content: content|default([]), html: html} only -%}
+
```

Comment:
The `with` and `block` are tied together so I'd suggest not putting space inbetween. (See also https://github.com/libero/pattern-library/pull/87/files#r281566133.)